python/tests/wrapper/test_01_synthetic_collection.py

```diff
@@ -0,0 +1,148 @@
+# This example generates a synthetic in-memory collection with 100 items and 60 tokens.
+# First 40 tokens are split on 10 clear topics according to this condition: "(token_id % 10) == (item_id % 10)"
+# Last 20 tokens are randomly allocated to documents.
+# The example demonstrates that in this clear situation BigARTM can precisely recover all topics.
+
+import artm.wrapper
+import artm.wrapper.messages_pb2 as messages
+import artm.wrapper.constants as constants
+import random
+import uuid
+import shutil
+import os
+import tempfile
+import itertools
+import pytest
```

Comment:
[PEP8: Imports](https://www.python.org/dev/peps/pep-0008/#id17)
Pay attention on how imports should be grouped and ordered